doc/docs/guides/releasing.md

```diff
@@ -0,0 +1,6 @@
+# Making a Release
+1. Tag the HEAD of the release branch and push to github: `git tag <version> && git push --tags`
+
+1. Check version is correct in `_version.py`.
```

Comment:
Are you deliberately writing a numbered list with all the numbers as 1. 1. 1. ?

Reply:
IIUC this is correct Markdown, any markdown preprocessor will auto-renumber a 1. 1. 1. 1. list as 1. 2. 3. 4, which makes it easier if you needed to re-order the items in the list ever.

I guess the downside is that anyone viewing the unpreprocessed source will not see a correctly numbered list.

Reply:
@pacharanero is correct here, it's how one does a markdown list for auto-numbering.  It's definitely a bit odd at first view but if you ever have to insert numbers into a list it's much easier!